lasy/profiles/transverse/hermite_gaussian_profile.py

```diff
@@ -10,31 +10,69 @@ class HermiteGaussianTransverseProfile(TransverseProfile):
     r"""
     A high-order Gaussian laser pulse expressed in the Hermite-Gaussian formalism.
 
+    Definition is according to Siegman "Lasers" pg. 646 eq. 60, as explicitly given
+    in https://doi.org/10.1364/JOSAB.489884 eq. 3,  with the beam center upon the
+    optical axis, :math:`x_0,y_0 = (0,0)`.
+
     More precisely, the transverse envelope (to be used in the
     :class:`.CombinedLongitudinalTransverseLaser` class) corresponds to:
 
     .. math::
-
         \mathcal{T}(x, y) = \,
-        \sqrt{\frac{2}{\pi}} \sqrt{\frac{1}{2^{n} n! w_0}}\,
-        \sqrt{\frac{1}{2^{n} n! w_0}}\,
-        H_{n_x}\left ( \frac{\sqrt{2} x}{w_0}\right )\,
-        H_{n_y}\left ( \frac{\sqrt{2} y}{w_0}\right )\,
-        \exp\left( -\frac{x^2+y^2}{w_0^2} \right)
+                \mathcal{H}_m (x) \, \mathcal{H}_n(y) \, \exp(i \Phi)
+
+    with
+
+    .. math::
+        \mathcal{H}_m(x) = A_m h_m \left ( \frac{\sqrt{2}x}{w_x(z)} \right) \exp{\left( -\frac{x^2}{w_x^2(z)}\right)} \exp{ \left ( -i k_0 \frac{x^2}{2 R_x(z)} \right )}
+
+        \mathcal{H}_n(y) = A_n h_n \left ( \frac{\sqrt{2}y}{w_y(z)} \right) \exp{\left( -\frac{y^2}{w_y^2(z)}\right)} \exp{ \left ( -i k_0 \frac{y^2}{2 R_y(z)} \right )}
+
+        w_x(z) = w_{0,x} \sqrt{1 + \left( \frac{z}{Z_x}\right)^2}
+
+        w_y(z) = w_{0,y} \sqrt{1 + \left( \frac{z}{Z_y}\right)^2}
+
+        A_m = \frac{1}{\sqrt{w_x(z) 2^{(m-1/2)} m!\sqrt{\pi}}}
+
+        A_n = \frac{1}{\sqrt{w_y(z) 2^{(n-1/2)} n!\sqrt{\pi}}}
+
+        R_x(z) = z + \frac{Z_x^2}{z}
+
+        R_y(z) = z + \frac{Z_y^2}{z}
+
+        \Phi(z) = \Phi_x(z) + \Phi_y(z)
+
+        \Phi_x(z) = \left(m+\frac{1}{2}\right) \tan^{-1}\left({\frac{z}{Z_x}}\right)
 
-    where  :math:`H_{n}` is the Hermite polynomial of order :math:`n`.
+        \Phi_y(z) = \left(n+\frac{1}{2}\right) \tan^{-1}\left({\frac{z}{Z_y}}\right)
+
+        \Z_x = \frac{\pi w_{0,x}^2}{\lambda_0}
+
+        \Z_y = \frac{\pi w_{0,y}^2}{\lambda_0}
+
+
+    where  :math:`h_{n}` is the Hermite polynomial of order :math:`n`.
+
+    The z-depedence shown in the above equations is required to correctly define the
+    electric field of the transverse profile relative to that of the pulse at the focus.
+    The absolute z position will be overwritten when creating a laser object.
 
     Parameters
     ----------
-    w0 : float (in meter)
-        The waist of the laser pulse, i.e. :math:`w_0` in the above formula.
-    n_x : int (dimensionless)
+    w_0x : float (in meter)
+        The waist of the laser pulse in the x direction,
+        i.e. :math:`w_{0,x}` in the above formula.
+    w_0y : float (in meter)
+        The waist of the laser pulse in the y direction,
+        i.e. :math:`w_{0,y}` in the above formula.
+    m : int (dimensionless)
         The order of hermite polynomial in the x direction
-    n_y : int (dimensionless)
+        i.e. :math:`m` in the above formula
```

Comment:
Is this required now that you propose to use the `m` letter consistently? Same thing for `n` below. 
```suggestion
```

Reply:
yes, I use m and n now rather than n_x and n_y